com.github.akiraux.interactive-goocanvas.json

```diff
@@ -0,0 +1,51 @@
+{
+   "app-id": "com.github.akiraux.interactive-goocanvas",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.30",
+   "sdk": "org.gnome.Sdk",
+   "base":"io.elementary.BaseApp",
+   "base-version": "juno",
+   "command": "com.github.akiraux.interactive-goocanvas",
+   "cleanup": [
+      "/include",
+      "/lib/pkgconfig",
+      "/lib/debug",
+      "/share/vala",
+      "/man",
+      "*.a",
+      "*.la"
+   ],
+   "finish-args": [
+      "--share=ipc",
+      "--socket=wayland",
+      "--share=network"
```

Comment:
What do we need network for here? 
Plus, does it not work with x11?

Reply:
I don't think we need Network access now. In the future for sure when we'll implement some sort of cloud syncing feature, but right now I can't think of any scenario where we need network access.

Regarding x11 or wayland, I've no idea so I let you guys decide :sweat_smile:

Reply:
fixed